File: pizza-api/pizza-data/models/Topping.cs
using System.Text.Json.Serialization;

namespace pizza_data.models
{
    public class Topping
    {
        public Topping() { }
        public Topping(int id, string name, string description, Pizza pizza) 
        {
            this.Id = id; this.Name = name; this.Description = description; this.PizzaId = pizza.Id;
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        [JsonIgnore]
        public int PizzaId { get; set; }
        [JsonIgnore]
        public virtual Pizza Pizza { get; set; }
    }
}

File: pizza-sequelize/models/topping.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const topping = sequelize.define('topping', {
    name: DataTypes.STRING,
    description: DataTypes.STRING,
    /*pizzaid: {
      type: DataTypes.INTEGER,
      references: {
         model: 'pizza',
         key: 'id',
      }
   }*/
  }, {});
  topping.associate = models => {
    topping.belongsTo(models.pizza, {
      foreignKey: {
        allowNull: false
      }
    });
    topping.hasMany(models.pizzaorderdetail, {
      onDelete: "cascade"
    });
  };
  
  return topping;
};